src/appmixer/mcpservers/chatmcp_server_perplexity_ask/CallTool/CallTool.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
 
         const inputs = {};
         tools.forEach(tool => {
-            if (tool.name === context.properties.tool && tool.inputSchema && tool.inputSchema.properties) {
+            if (tool.name === context.properties.tool?.inputSchema && tool.inputSchema.properties) {
```

Comment:
This fix is not correct!

It should be

```
if (tool.name === context.properties.tool && tool.inputSchema?.properties) {
```

The same problem is in all `src/appmixer/mcpservers/**/CallTool/CallTool.js` files